docker.nix

```diff
@@ -0,0 +1,84 @@
+{ dockerTools
+, backend
+, frontend
+, linkFarm
+, runCommand
+, writeTextDir }:
+
+let
+  inherit (dockerTools) buildLayeredImage buildImage pullImage;
+in
+
+{
+  backend-image = let
+    # linkFarm seems to put a dangling symlink in the image when used here, so
+    # we use runCommand and copy the file instead.
+    init-sql = runCommand "edna-init-sql" {} ''
+      mkdir -p $out
+      cp ${./backend/sql/init.sql} $out/init.sql
```

Comment:
Minor suggestion: maybe let's put it into `$out/sql/init.sql` so that `dev-config.yaml` works? In `dev-config.yaml` we have `init-script: "./sql/init.sql"`.